File: problems/95/Main.py
from __future__ import print_function
import math

#--SPLIT--
#-class Solution:
	# @param A : list of integers
	# @return a list of integers
#-	def maxset(self, A):

#--SPLIT--
class Solution:
	def maxset(self, A):
		return A


#--SPLIT--

if __name__ == "__main__":
    test_cases = int(raw_input())
    for i in range(test_cases):
        A = [int(l) for l in raw_input().split(" ")][1:]
        solution = Solution()
        result = solution.maxset(A) 
        for r in result:
            print(str(r) + " ", end="")
        print("")

File: problems/95/Main.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
#include<string.h>

//--SPLIT--

//-int* plusOne(int* A, int n) {
//-}

//--SPLIT--
int* maxset(int* A, int n) {
	return 0;
}

//--SPLIT--
int main() {
	setbuf(stdout, NULL);
	int testcases;
	scanf("%d", &testcases);
	int t = 0;
	for (t = 0; t < testcases; t++) {
		int n, i;
		scanf("%d", &n);
		int *A = malloc (sizeof (int) * n);
		for (i = 0; i < n; i++)  {
			scanf("%d", &A[i]);
		}
		int * result = maxset(A, n);
		for (i = 0; i < n; i++) {
			printf("%d ", result[i]);
		}
		printf("\n");
	}
	return 0;
}
